homeassistant/components/ld2410_ble/sensor.py

```diff
@@ -0,0 +1,147 @@
+"""LD2410 BLE integration sensor platform."""
+
+
+from collections.abc import Callable
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfLength
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import LD2410BLE, LD2410BLECoordinator
+from .const import DOMAIN
+from .models import LD2410BLEData
+
+distance_description: Callable[
```

Comment:
Let's not use a factory function. You're not saving many extra lines but you're making it harder to read.

Reply:
Also, it seems like the `state_class` is missing?

Reply:
> Let's not use a factory function. You're not saving many extra lines but you're making it harder to read.

It'll save over 100 lines in the next MR, which will make it easier to read, but I've reverted it for now so we can discuss it when that time comes.

> Also, it seems like the `state_class` is missing?

Fixed.